### DISCUSSION TITLE
Why does my Flood have ADHD - Spamming new notifications on tab

### DISCUSSION THREAD

**Discussion by ghost** [jesec/flood#425] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/30607152/136574654-f042ae92-10c6-433a-8ad3-0042baeb05dc.png)

Flood is constantly having this notification symbol on my tab, while not having any notifications on the side (cleared them all). When being on the web UI the notification goes away but as soon as I leave the tab, it soon becomes the picture above. This is a problem because I have this tab in a folder for all my home server tabs and thus I can not see if there really is a notification.

![image](https://user-images.githubusercontent.com/30607152/136575004-d36c6d4a-a45d-4444-afec-3575ea78f7ef.png)

**Comment by jesec** [comment, marked as answer]
#395
